.changelog/542.txt

```diff
@@ -0,0 +1,3 @@
+```release-note:improvement
+Capture unknown error from SDK when getting project or organization.
```

Comment:
```suggestion
```release-note:bug
Capture unknown errors from the HCP Go SDK when getting project or organization, where errors were previously replaced with a static message.
```